lib/workers/global/index.ts

```diff
@@ -123,12 +124,13 @@ export async function start(): Promise<number> {
   }
 
   let config: AllConfig;
+  const env = getEnv();
```

Comment:
is the global env already configured here?

Reply:
The only possible env available at this moment would be from `process.env`